packages/shared/src/services/__tests__/ClaudeStreamingService.test.ts

```
@@ -0,0 +1,698 @@
+            }
+            return session!;
+          }),
+        
+        closeSession: (sessionId: string) =>
+          Effect.gen(function* () {
+            const session = sessions.get(sessionId);
+            if (session) {
+              session.isActive = false;
+              session.lastActivity = Date.now();
+            }
+          }),
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)